primary/src/core.rs

```diff
@@ -539,7 +580,6 @@ impl Core {
             .cleanup(self.committee.network_diff(&committee));
 
         // Cleanup internal state.
-        self.last_voted.clear();
```

Comment:
Although we are replacing the `last_voted` map with the storage, we should still clear the `vote_store` 
 contents when we change epoch (since we start again from round 0), otherwise we won't be able to vote on any header and that will kill liveness